### PR TITLE
updated privy and txkit deps, added logout, enabled testnets

### PR DESCRIPTION
**Pull request by poocart**
## Description

- Updated `@privy-io/react-auth` to `1.54.3`
- Updated `@etherspot/transaction-kit` to `0.7.0`
- Added `viem` utils for Privy supported `Chain` type
- Fixed mainnet and testnet native chain assets
- Added `@etherspot/transaction-kit` Etherspot `projectKey` support
- Added `REACT_APP_USE_TESTNETS` env var to enable testnets by default (default testnet chain – Goerli)

## How Has This Been Tested?

- Locally sent 0.001 ETH on Goerli and ran JS tests.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "pillarx",
-  "version": "0.1.10",
+  "version": "0.1.11",
```
Comment (IAmKio) [inline package.json:3]: Definitely a minor bump - we've introduced features 🙏

File: src/containers/Main.tsx [hunk 2]
```diff
@@ -84,6 +90,7 @@ const Main = () => {
           appId={process.env.REACT_APP_PRIVY_APP_ID as string}
           config={{
             appearance: { theme: 'dark' },
+            defaultChain: process.env.REACT_APP_USE_TESTNETS === 'true' ? goerli : mainnet,
```
Comment (IAmKio) [inline src/containers/Main.tsx:93]: Given that Goerli is sunset should we use this opportunity to move to Sepolia? Will need to check actually if Prime SDK supports this

Comment (IAmKio) [reply src/containers/Main.tsx:93]: Yes it's supported: https://sdk.etherspot.io/enums/networknames#sepolia

File: src/utils/blockchain.tsx [hunk 3]
```diff
@@ -13,63 +13,77 @@ export const isValidEthereumAddress = (address: string | undefined): boolean =>
...
-  [MAINNET_CHAIN_ID.ETHEREUM_MAINNET]: {
-    chainId: MAINNET_CHAIN_ID.ETHEREUM_MAINNET,
+/**
+ * Cross-check with:
+ * - https://etherspot.fyi/prime-sdk/chains-supported
+ * - https://docs.privy.io/guide/configuration/networks#default-configuration
+ */
+export const TESTNET_CHAIN_ID = {
+  POLYGON_MUMBAI: 80001,
+  SEPOLIA: 11155111,
```
Comment (IAmKio) [inline src/utils/blockchain.tsx:40]: Ha great chain ID

**Review by IAmKio — CHANGES_REQUESTED** [review]
Just a few minor comments but other than that - all good

File: package.json (hunk 1 above)
Comment (poocart) [reply package.json]: makes sense, changed to `0.2.0`

File: src/utils/blockchain.tsx (hunk 3 above)
Comment (poocart) [reply src/utils/blockchain.tsx]: HAHAH wen 69420 chain

File: src/containers/Main.tsx (hunk 2 above)
Comment (poocart) [reply src/containers/Main.tsx]: makes sense, changed

**Review by IAmKio — APPROVED** [review]
LGTM

**Pull request merged**